terraform/app/terraform.tf

```diff
@@ -0,0 +1,48 @@
+/*
+For username / password authentication:
+- user
+- password
+For SSO authentication
+- sso_passcode
+- store_tokens_path = /path/to/local/file
+*/
+
+provider cloudfoundry {
+  api_url           = var.paas_api_url
+  password          = var.paas_password != "" ? var.paas_password : null
+  sso_passcode      = var.paas_sso_passcode != "" ? var.paas_sso_passcode : null
+  store_tokens_path = "./tokens"
+  user              = var.paas_user != "" ? var.paas_user : null
+
+}
+
+/*
+Store infrastructure state in a remote store (instead of local machine):
+https://www.terraform.io/docs/state/purpose.html
+*/
+# terraform {
+
+#   backend "s3" {
+#     key     = "terraform.tfstate"
+#     region  = "eu-west-2"
+#     encrypt = "true"
+#     bucket    = "eyfs-terraform-backend"
```

Comment:
Probs put the bucket name in a secret

Reply:
This is not required for now - I'll remove